llmcode/persistence_layer.py

```
@@ -0,0 +1,1145 @@
+            
+            # Scan index files
+            for index_file in self.storage_dir.glob("indexes/*/*.idx"):
+                try:
+                    with open(index_file, 'r') as f:
+                        index_data = json.load(f)
+                    
+                    if analysis_type is None or index_data.get('analysis_type') == analysis_type:
+                        cache_keys.append(index_data['cache_key'])
+                        
+                except Exception:
+                    continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)